README.md

```diff
@@ -116,6 +116,16 @@ Everything you need is available prebuild in FreeBSD' ports.
 
     pkg install gmake node
 
+#### Arch:
+
+You can install cjdns by running
+
+    pacman -S cjdns
+
+If you need to build from source, everything you need can be installed like this
+
+    pacman -S nodejs git base-devel
```

Comment:
Why not point to AUR package https://aur.archlinux.org/packages/cjdns-git/ via pacaur like `pacaur -S cjdns-git`?